ee/src/main/java/org/jboss/as/ee/logging/EeLogger.java

```diff
@@ -1134,4 +1135,16 @@
 
     @Message(id = 117, value = "Field %s cannot be set - object of %s loaded by %s is not assignable to %s loaded by %s")
     IllegalArgumentException cannotSetField(String fieldName, Class<?> injectedClass, ClassLoader injectedClassloader, Class<?> fieldClass, ClassLoader fieldClassloader);
+
+    @LogMessage(level = INFO)
+    @Message(id = 118, value = "The system property 'ee8.preview.mode' is set to 'true'. For provided EE 8 APIs where the EE 8 " +
+            "version of the API differs from what is supported in EE 7, the EE 8 variant of the API will be used. " +
+            " Support for this setting will be removed once all EE 8 APIs are provided and certified.")
```

Comment:
Not a major issue, but this has a double space between "used." and "Support".

Reply:
@jamezp Sadly, the double spacing issue is all over the codebase... just a simple grep:

```
[rhusar@ribera wildfly]$ git grep "\.\ \ " | grep Logger
ee/src/main/java/org/jboss/as/ee/logging/EeLogger.java:    @Message(id = 83, value = "%s injection target is invalid.  Only setter methods are allowed: %s")
ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:    @Message(id = 462, value = "Unable to detect database dialect from connection metadata or JDBC driver name. Please configure this manually using the 'datasource' property in your configuration.  Known database dialect strings are %s")
ejb3/src/main/java/org/jboss/as/ejb3/timerservice/persistence/database/DatabaseTimerPersistence.java:                EjbLogger.EJB3_TIMER_LOGGER.debugf("Detect database dialect as '%s'.  If this is incorrect, please specify the correct dialect using the 'database' attribute in your configuration.  Supported database dialect strings are %s", database, databaseDialects);
jaxrs/src/main/java/org/jboss/as/jaxrs/logging/JaxrsLogger.java:    @Message(id = 15, value = "No Servlet declaration found for JAX-RS application.  In %s either provide a class that extends javax.ws.rs.core.Application or declare a servlet class in web.xml.")
jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/logger/JdrLogger.java://    @Message(id = 6, value = "Plugin contrib location is not a directory.  Ignoring.")
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:        "in application.  One of the duplicate persistence.xml should be removed from the application." +
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:        " Application deployment will continue with the persistence.xml definitions from %s used.  " +
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java://    @Message(id = 5, value = "Could not load default persistence provider adaptor module.  Management attributes will not be registered for the adaptor")
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:    @Message(id = 6, value = "Could not load default persistence provider module.  ")
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:            "because the transaction already has a transactional context associated with it.  " +
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:            "persistence context or the transactional context.  See JPA spec 2.0 section 7.6.3.1.  " +
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:    @Message(id = 59, value = "%s injection target is invalid.  Only setter methods are allowed: %s")
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:     * were multiple persistence unit definitions in the app.  JBoss 5 loosened the checking up, to let applications,
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:     * just use any PU definition that they find.  For AS7, we are strictly enforcing this again just like we did in
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:            "but a component with a 'SynchronizationType.SYNCHRONIZED' is now being used.  " +
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:            "change the called component code to also use 'SynchronizationType.UNSYNCHRONIZED'.  "+
jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java:            "See JPA spec 2.1 section 7.6.4.1.  " +
jsf/subsystem/src/main/java/org/jboss/as/jsf/logging/JSFLogger.java:    @Message(id = 5, value = "Unknown JSF version '%s'.  Default version '%s' will be used instead.")
messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/PropertySQLProviderFactory.java:                MessagingLogger.ROOT_LOGGER.debugf("Detect database dialect as '%s'.  If this is incorrect, please specify the correct dialect using the 'database' attribute in your configuration.  Supported database dialect strings are %s", database, databaseDialects);
messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/logging/MessagingLogger.java:    @Message(id = 94, value = "Unable to detect database dialect from connection metadata or JDBC driver name. Please configure this manually using the 'journal-database' property in your configuration.  Known database dialect strings are %s")
naming/src/main/java/org/jboss/as/naming/logging/NamingLogger.java:    @Message(id = 23, value = "Duplicate JNDI bindings for '%s' are not compatible.  [%s] != [%s]")
naming/src/main/java/org/jboss/as/naming/logging/NamingLogger.java:    @Message(id = 32, value = "Invalid context reference.  Not a '%s' reference.")
security/subsystem/src/main/java/org/jboss/as/security/logging/SecurityLogger.java:    @Message(id = 104, value = "Default %s cache capability missing.  Assuming %s as default-cache.")
webservices/server-integration/src/main/java/org/jboss/as/webservices/logging/WSLogger.java:    @Message(id = 48, value = "@WebServiceRef injection target is invalid.  Only setter methods are allowed: %s")
```

Reply:
Yeah. I just happened to notice, but I really don't think it's that big of a deal.